src/app/learn/get-started/advanced/accessibility-analyzer/index.html

```diff
@@ -4,6 +4,9 @@
   showTableOfContents="true">
   
   <stache-page-summary>
+  <sky-alert alertType="warning">
+    <stache-include fileName="e2e-recommendation.html"></stache-include> If you must use e2e tests, the deprecated accessibilty analyzer described here lets you add accessibility checks to e2e specs and manage rules for accessibility If not, we recommend <a routerLink="/learn/get-started/advanced/accessibility-unit-tests">running accessibility checks in SKY UX unit tests</a>.
```

Comment:
- "deprecated accessibilty analyzer" Misspelled
- "rules for accessibility If not" Should there be a period here?

Reply:
Corrected the spelling of "accessibility" (Nice catch!) and added the missing period.